src/util/ChartUtils.ts

```diff
@@ -32,6 +32,11 @@ import {
   ChartOffset,
 } from './types';
 import { getLegendProps } from './getLegendProps';
+import { Props as XAxisProps } from '../cartesian/XAxis';
+import { Props as YAxisProps } from '../cartesian/YAxis';
+import { Props as ZAxisProps } from '../cartesian/ZAxis';
+
+type CartesianAxisProps = XAxisProps | YAxisProps | ZAxisProps;
```

Comment:
😅 generally I'd put this below the re-export below but all good

Reply:
At the line 43?